sdk/communication/Azure.Communication.Chat/src/ChatClientOptions.cs

```diff
@@ -25,7 +25,7 @@ public ChatClientOptions(ServiceVersion version = LatestVersion)
         {
             ApiVersion = version switch
             {
-                ServiceVersion.V1 => "2020-11-01-preview3",
+                ServiceVersion.V1 => "2021-01-27-preview4",
```

Comment:
`V1` name in the enum also needs to be renamed to `V2021_01_27_Preview4". This is a feedback that we recently got for other packages

Reply:
Do you mean to change from       
```
public enum ServiceVersion
        {
            V1 = 1
        }
```
to 
  
```
public enum ServiceVersion
        {
            V1 =  'V2021_01_27_Preview4'
        }
```
?

Reply:
See this: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClientOptions.cs#L36

Reply:
changed :)